### ISSUE TITLE
Bug: Content script not working in 0.17.0 but works in 0.16.11

### ISSUE THREAD

**Issue by mefengl** [wxt-dev/wxt#481]
### Describe the bug

The code in the main function of `defineContentScript` doesn't run since 0.17.0

### To Reproduce

Share minimal reproduction. Examples of a minimal reproduction:

0.16.11 ✅: https://github.com/mefengl/wxt-examples/commit/b92270640fee834adce35c7de845d835f2addec0
0.17.0 ❌: https://github.com/mefengl/wxt-examples/commit/060cf9c7e30f0d5fcfc715ae68ead8d5cf247032

all changes: https://github.com/wxt-dev/wxt-examples/compare/main...mefengl:wxt-examples:main

Steps to reproduce the bug using the reproduction:

1. Install dependencies: `pnpm i`
2. Run: `cd examples/react`
3. Start dev mode: `pnpm dev`
4. Open any website
5. Check the developer tools' console

### Expected behavior

Not applicable

### Screenshots

<img width="626" alt="image" src="https://github.com/wxt-dev/wxt/assets/71683364/86adaffd-e7cd-4b9c-88a1-807ed7a3845b">

### Environment

```
  System:
    OS: macOS 14.1
    CPU: (8) arm64 Apple M1
    Memory: 49.89 MB / 8.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.10.0 - ~/.nvm/versions/node/v20.10.0/bin/node
    Yarn: 1.22.19 - ~/.pnpm-global/bin/yarn
    npm: 10.4.0 - ~/.nvm/versions/node/v20.10.0/bin/npm
    pnpm: 8.9.0 - /opt/homebrew/bin/pnpm
    bun: 1.0.15 - ~/.pnpm-global/bin/bun
    Watchman: 2024.01.22.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 122.0.6261.69
    Edge: 121.0.2277.128
    Safari: 17.1
```

**Comment by aklinker1** [comment]
Oh woops, thanks for reporting. I know what's causing this.

**Comment by aklinker1** [comment]
Fixed in [`v0.17.1`](https://github.com/wxt-dev/wxt/releases/tag/v0.17.1)
